CSVParser/CSVFile.cs

```diff
@@ -1,12 +1,16 @@
-﻿using System;
+﻿/* CSVFile.cs
+ * Author: Hunter Bennett, Connor Black, James Dunton
```

Comment:
Can you just tab the authors too to line up with the desc that is tabbed below?

Reply:
All tabbed now